docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -1032,3 +1045,37 @@ like a normal view, and the data is queried directly from the base tables.
 
 Dropping a materialized view with :doc:`/sql/drop-materialized-view` removes
 the definition and the storage table.
+
+Table statistics
+----------------
+
+There is experimental support to collect column statistics which can be enabled by
+setting the ``iceberg.experimental.extended-statistics.enabled`` catalog
+configuration property or the corresponding
+``experimental_extended_statistics_enabled`` session property to ``true``.
+Enabling this configuration allows executing :doc:`/sql/analyze` statement to gather statistics.
+
```

Comment:
We have to add if it is necessary to run analyse or if stats are also collected and/or update on the fly with data changes (insert, update, delete..)